### PR TITLE
Use persistant trace functionality from dynawo to properly catch all …

### PR DESCRIPTION
**Pull request by rosiereflo**
…traces

### CODE REVIEW

File: sources/main.cpp [hunk 1]
```diff
@@ -116,35 +116,43 @@ main(int argc, char* argv[]) {
     context.execute();
 
     // Traces must be re-initiliazed to append to current DynaflowLauncher log file as simulation had modified it
-    dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
     LOG(info) << MESS(SimulationEnded, context.basename(), timerSimu.elapsed()) << LOG_ENDL;
     LOG(info) << " ============================================================ " << LOG_ENDL;
     LOG(info) << MESS(DFLEnded, context.basename(), timerGlobal.elapsed()) << LOG_ENDL;
+    DYN::Trace::resetPersistantCustomAppenders();
```
Comment (lecourtoisflo) [inline sources/main.cpp:122]: Why perform a reset here since the program ends ?

File: sources/main.cpp [hunk 2]
```diff
@@ -116,35 +116,43 @@ main(int argc, char* argv[]) {
     context.execute();
 
     // Traces must be re-initiliazed to append to current DynaflowLauncher log file as simulation had modified it
-    dfl::common::Log::init(options, config.outputDir(), continuePreviousLogFile);
```
Comment (lecourtoisflo) [inline sources/main.cpp:119]: Maybe modify Log::init function since we no longer need the continuePreviousLogFile argument

**Review by lecourtoisflo — CHANGES_REQUESTED** [review]

**Review by marcochiaramello — APPROVED** [review]

File: sources/main.cpp (hunk 1 above)
Comment (rosiereflo) [reply sources/main.cpp]: solved

File: sources/main.cpp (hunk 2 above)
Comment (rosiereflo) [reply sources/main.cpp:119]: solved

**Review by lecourtoisflo — APPROVED** [review]

**Review by PerninGuillaume — APPROVED** [review]

**Review by gautierbureau — APPROVED** [review]

**Pull request merged**